packages/scrum-client/src/context/context.js

```diff
@@ -22,7 +22,7 @@ export function useAuthDispatch() {
   return context;
 }
 
-export const AuthProvider = ({ children }) => {
+export const AuthProvider = ({ children, initialState = authInitialState }) => {
```

Comment:
As a common improvement , try to avoid any external dependency , it will make it harder to test later on, so  now initialState can be passed via props or have a default value.